src/emqx_web_hook_actions.erl

```diff
@@ -37,7 +37,7 @@
                          description => #{en => <<"Request Header">>,
                                           zh => <<"请求头"/utf8>>}},
             method => #{type => string,
-                        enum => [<<"PUT">>,<<"POST">>],
+                        enum => [<<"PUT">>,<<"POST">>,<<"GET">>,<<"DELETE">>],
                         default => <<"POST">>,
                         title => #{en => <<"Request Method">>,
                                    zh => <<"请求方法"/utf8>>},
```

Comment:
We should add a note in the description I think:
```
description => #{en => <<"Request Method. Note that the payload_template will be discarded in case of GET method">>,
                           zh => <<"请求方法。注意：当请求方法为 GET 的时候，payload_template 参数会被忽略"/utf8>>}}
```

Reply:
added.

Reply:
added.